.jazzy.yaml

```diff
@@ -10,15 +10,3 @@ module: VelocidiSDK
 output: ./docs/docs_page/
 theme: docs/velocidi-jazzy-theme/
 readme: ./docs/tmp/readme/README.md
```

Comment:
Why is this required?

Reply:
There has to be some kind of landing page. Thought about removing it, but it would default to just using the whole README as main page.